### PR TITLE
Draft all merchandising docs

### PR DESCRIPTION
**Pull request by eballantine**
So I feel I'm definitely missing some key things, and some of this is probably more the Creator/Success audience, so feel free to critique!

### CODE REVIEW

File: docs/product_management_and_merchandising/accommodation_and_experience_products.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Accommodation and Experience Products
+parent: Product Management and Merchandising
+nav_order: 3
+---
+
+Accommodation type products have one key difference which is a Daily Availability & Pricing feature. This feature expects what is being sold to be available across many dates, and that the customer should be able to choose check in and check out dates, perhaps with a minimum nights' stay.
```
Comment (paulina-a) [inline docs/product_management_and_merchandising/accommodation_and_experience_products.md:8]: Perhaps there should be a quick line just saying Products can be either 'experiences' or 'accommodations'?

File: docs/product_management_and_merchandising/variants.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{% endraw %}
+
+Might render:
+{% raw %}
+```html
+<h2>Double Rooms</h2>
+<ul>
+    <li>Winter Wonderland Room</li>
+    <li>Bears Den Room</li>
+    <li>Log Cabin Room</li>
```
Comment (paulina-a) [inline docs/product_management_and_merchandising/variants.md:36]: I would like one 'Bears den room' please

File: docs/product_management_and_merchandising/extras.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+</ul>
+
+<h2>Single Rooms</h2>
+<ul>
+    <li>Winter Wonderland Room</li>
+    <li>Bears Den Room</li>
+    <li>Log Cabin Room</li>
+</ul>
+```
+{% endraw %}
```
Comment (paulina-a) [inline docs/product_management_and_merchandising/extras.md:47]: Update references to 'variant' with 'extra'

File: docs/product_management_and_merchandising/useful_info.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Useful Info
+parent: Product Management and Merchandising
+nav_order: 10
+---
+Product's can have details added to **Settings > Userful info**.
```
Comment (paulina-a) [inline docs/product_management_and_merchandising/useful_info.md:7]:
``` suggestion
Products can have details added to **Settings > Userful info**.

**Review by paulina-a — APPROVED** [review]
Nice work, that was a big chunk to tackle!

File: docs/product_management_and_merchandising/extras.md (hunk 3 above)
Comment (eballantine) [reply docs/product_management_and_merchandising/extras.md]: Yep 🙈 not sure how that happened, I got halfway through and lost the plot maybe

**Pull request merged**